reo/views.py

```diff
@@ -709,6 +709,8 @@ def chiller_defaults(request):
                         absorp_chiller_opex = defaults_sizes[size - 1][2] + slope_opex * \
                                               (max_cooling_load_tons - defaults_sizes[size - 1][0])
 
+        absorp_chiller_elec_cop = 14.1
```

Comment:
@Bill-Becker everything looks good. My only concern is that this hard-coded value won't update when someone updates the default value in `nested_inputs.py`. I checked and `nested_input_definitions` is already imported into `views.py`, so I think that you can get the value from the `nested_input_definitions` dict?

Reply:
@NLaws Thanks for the review. I linked this value to the `nested_input_definitions` default value for this in [this commit](https://github.com/NREL/REopt_Lite_API/pull/158/commits/ca0df333322943317d39d04da14968c9e00d05af).